inc/RestApi/SettingsController.php

```diff
@@ -175,7 +175,7 @@ public function get_current_settings() {
 		}
 
 		$settings = array(
-			'comingSoon'              => container()->get( 'comingSoon' )->is_enabled(),
+			'comingSoon'              => container()->has( 'comingSoon' ),
```

Comment:
This changes the meaning behind the `comingSoon` property. The `has()` check just looks to see if the service is registered in the container. We should look into why the coming soon service doesn't exist yet and fix that.

Reply:
Yeah, I wrote that first then realised there was more going on than i quite understood.